Lib/genericpath.py

```diff
@@ -122,8 +123,7 @@ def commonprefix(m):
 # describing the same file?
 def samestat(s1, s2):
     """Test whether two stat buffers reference the same file"""
-    return (s1.st_ino == s2.st_ino and
```

Comment:
I propose to exclude stylistic-only changes. But, module maintainers might think otherwise (maybe these modules do need a refactor?).

But, generally style-only changes are not really helpful.

Reply:
Do you see these other changes as _stylistic-only_?

- https://docs.astral.sh/ruff/rules/multiple-statements-on-one-line-colon: https://github.com/python/cpython/blob/c2c04bf40f49cb3960e4396a872b06e6e7eb1533/Lib/genericpath.py#L107-L108
- Not combining as imports (default isort setting): https://github.com/python/cpython/blob/c2c04bf40f49cb3960e4396a872b06e6e7eb1533/Lib/ntpath.py#L46-L48
- https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator: https://github.com/python/cpython/blob/c2c04bf40f49cb3960e4396a872b06e6e7eb1533/Lib/ntpath.py#L322-L325

Reply:
> Do you see these other changes as _stylistic-only_?

yes